docs/css/ez-guidelines-form-components.css

```diff
@@ -0,0 +1,62 @@
+.ez-guidelines .ez-code-example .ez-label {
+    margin-bottom: 0;
+    font-size: 1em;
+    font-weight: 700;
+    color: #333;
+}
+.ez-guidelines .ez-code-example .ez-label:after {
+    content: ': ';
+}
+.ez-guidelines .ez-code-example .form-group .form-control {
+    display: block;
+    width: 100%;
+    height: 38px;
+    padding: .375em .75em;
+    font-size: 1em;
+    line-height: 1.5;
+    color: #878787;
+    background-clip: padding-box;
+    border: 1px solid #ededed;
+    border-radius: .25em;
+    transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
```

Comment:
In our code we're using `cubic-bezier(0.25, 0.8, 0.25, 1)` as a transition function. Should we change it? @inakijv